### PR TITLE
Basic working docs. Fixes #4

### LINKED ISSUE #4: Sphinx documentation to RTD
There's not much beyond what can be auto-generated, but it's a start and will allow for intersphinx-links.

### PR DESCRIPTION
**Pull request by jamadden**
This is all pretty trivial scaffolding and doesn't touch the code or existing docstrings at all. 

In the future it would be good to have some info about configuration and keyreferences.

Until this is merged to master you can view the results at http://zopeintid.readthedocs.io/en/docs/

### CODE REVIEW

**Comment by jamadden** [comment]
Are there any interested reviewers? This one is pretty trivial (no code changes) and I will probably merge soon if there are no reviews/objections.

File: README.rst [hunk 1]
```diff
@@ -4,7 +4,13 @@
 .. image:: https://travis-ci.org/zopefoundation/zope.intid.png?branch=master
         :target: https://travis-ci.org/zopefoundation/zope.intid
 
+.. image:: https://readthedocs.org/projects/zopeintid/badge/?version=latest
+         :target: http://zopeintid.readthedocs.io/en/latest/?badge=latest
+         :alt: Documentation Status
+
 This package provides an API to create integer ids for any object. Later
 objects can be looked up by their id as well. This functionality is commonly
 used in situations where dealing with objects is undesirably, such as in
```
Comment (davisagli) [inline README.rst]: undesirable

File: doc/index.rst [hunk 2]
```diff
@@ -0,0 +1,41 @@
+=====================
+ What is zope.intid?
+=====================
+
+This package provides an API to create integer ids for any object. Later
+objects can be looked up by their id as well. This functionality is commonly
+used in situations where dealing with objects is undesirably, such as in
```
Comment (davisagli) [inline doc/index.rst]: undesirable

**Review by davisagli — APPROVED** [review]
LGTM aside from this (already existing) typo.

**Comment by jamadden** [comment]
Thanks for the review! I plan a follow up PR to add some additional user-level documentation.

**Pull request merged**